### ISSUE TITLE
[Question/Inconsistent behaviour] Window Size + Video Filter -> Windows Vs. Linux version

### ISSUE THREAD

**Issue by LukasThyWalls** [TASEmulators/desmume#612]
Hello.

I know the Linux version is a bit back to the Windows version, but the behaviour with the "Window Size" + "Video Filter" options is a bit different between both and the "newer" (Windows) one doesn't seem right.

I'm using the Linux version i'm using is _0.9.12 git#0 x64-JIT SSE2_ (the Flatpak one), and the Windows versions tested is 0.9.13 x64-SSE2 and the nightly build 0.9.14 git#91efef9 x64-JIT SSE2, both ones doing the same behaviour.

In both you can choose the "Window Size": 1x, 1.5x, 2x, 2.5x, 3x, 4x, 5x and Scale to Window. The Linux version has also 0.5x as an option, which is weird. In "Video Filter" there are tons of video filters, SuperEagle, Scanline, BRZ, LQ, HQ, Nearest... with some with multipliers 15 (1.5), 2, 3...

The problem is, in the Linux version you can set a Window Size and a filter with a multiplier, and it keeps the Window size applying the selected Video Filter. However, in the Windows version when you select a Video Filter with a multiplier, it applies the multiplier to the Window Size, making the window bigger.

So thinking that the Windows version is _newer_ than the Linux one, i thought it's a new intended behaviour, but it introduces the issue that most of the Windows Size are unusable unless you are using a 4K monitor. The better Window Size option to use with 1080p with the emulator in vertical layout is 2.5x, and in Windows can't be used because any of the Video Filter multipliers that are 2x or more are multiplied to the Window Size, when in Linux, you can use a Window Size of 2.5x and the 3xBRZ Video Filter without any problem.

To summarize, **the Window Size is the absolute size of the Window independent of the Video Filter multiplier (Like the 0.9.12 Linux Version) or it is a relative Window Size joined with the Video Filter multiplier (Like the 0.9.13/nightly Windows versions)?**

Thanks in advance!

**Comment by zeromus** [comment]
I'm not going to read all this. Your assumptions are all wrong and you're overthinking it from flawed assumptions. It doesn't seem like you're having a problem, because at the end as a kind of summary you put a question in bold. You can answer the question yourself with some experimentation. The question concerns, I think, some "version" of desmume, which I would have to decipher from the wall of text I'm not going to read. However since you're asking it to be compared to the latest version, I assume you're asking about the not-latest version, in which case my answer is I don't know and I don't care. Use the latest version and tell me if you have a problem.

**Comment by LukasThyWalls** [comment]
Thanks
